requirements.bazel.txt

```diff
@@ -38,3 +38,5 @@ typing-extensions==4.9.0
 pyasn1-modules==0.3.0
 zipp==3.17.0
 wrapt==1.16.0
+xds-protos==0.0.11
```

Comment:
As stated in previous PR reviews, this dependency is defined in this repo and should be pulled in hermetically, not via PyPi.